internal/distro/rhel7/distro.go

```diff
@@ -356,6 +358,13 @@ func (t *imageType) initializeManifest(bp *blueprint.Blueprint,
 		w.DisabledServices = services.Disabled
 	}
 
+	if t.workload != nil {
+		w.Packages = append(w.Packages, t.workload.GetPackages()...)
```

Comment:
I wonder if these 4 lines should be factored out into a `Workload` method. Would help a bit with duplicated code.

Reply:
We might drop this entirely and not allow any customisations for workload-specific images.  We discussed this a bit with @teg but haven't made a long-term plan yet.
The current plan is to restrict this image type to Koji and block any customisations for this image.

Reply:
FWIW, I think that not allowing customizations for certain types of images goes against the added value of image builder to allow building customized images. In any case, I would like us to have a wider discussion before restricting things.

Reply:
I'd like to discuss it too.  I'm wondering whether it's safer to not allow any customisation to begin with and maybe enable some later rather than the other way around.  Maybe it doesn't matter for image types that are (for now) internal only.

Reply:
So I have thoughts and plans about the workloads and customizatons.  It's all a priority for this quarter, so I'm going to plan it out and we can talk about the details.  But for now, I think it's safer and quicker to make customizations and non-custom workloads mutually exclusive, which I did now in this PR.  The image type is internal only now, so there _shouldn't_ be an issue blocking customizations.